src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -307,15 +309,27 @@ const Inventory = ({
 
       systemProfile = {
         ...systemProfile[0],
-        transforms: [wrappable],
+        transforms: [sortable, wrappable],
         props: { isStatic: true },
       };
 
       tags = {
         ...tags[0],
       };
 
-      let columnList = [displayName, tags, systemProfile, lastSeenColumn];
+      groups = {
+        ...groups[0],
+        transforms: [wrappable],
+        props: { isStatic: true },
```

Comment:
We need to make sure that column props are extended with `isStatic: true` instead of being mutated fully. Because there are other column props coming from Inventory to give the column the correct width, etc.

Reply:
Good catch, I didn't notice that!  Done!